### DISCUSSION TITLE
In-line inputs

### DISCUSSION THREAD

**Discussion by alexisgauba** [microsoft/vscode-discussions#681] [Extension Development QnA] [answered]
Is there a way to create in-line inputs in the editor with the API for extensions?

I know there is QuickPick and InputBox, but seems like neither of those appear in-line. Is there a context input like there is a context menu for users to be able to add inputs near the exact code that is relevant to the input?

**Comment by alefragnani** [comment, marked as answer]
Hi @alexisgauba ,

There is no API available yet, but there is an API proposal on backlog https://github.com/microsoft/vscode/issues/85682.

The recent release of GitHub Copilot provides such element as recently commented on that issue, so I suppose a similar API could available in the future, just like the inline completion API was.

Hope this helps
